tests/strategy/strats/SampleStrategy/__init__.py

```diff
@@ -0,0 +1 @@
+../../../../freqtrade/templates/sample_strategy.py
```

Comment:
This is definitely wrong. 

A package file `__init__.py` cannot contain a relative path - but must import a module to work correctly.

Reply:
This is what symbolic links look like in git

Reply:
Shocking - that's ... very irritating from github's side ...